libs/blocks/video/video.js

```diff
@@ -1,13 +1,30 @@
 export default function init(a) {
   const { pathname, hash } = a;
-
   const isAutoplay = !!(hash?.includes('autoplay'));
   const isNotLooped = !!(hash?.includes('autoplay1'));
-
-  const attrs = isAutoplay ? `playsinline autoplay ${isNotLooped ? '' : 'loop'} muted` : 'playsinline controls';
+  const playOnHover = !!(hash.includes('hoverplay'));
```

Comment:
nit: all of the `.includes` calls do not need to be prepended by `!!` since they return bool values